examples/restaurantbot/run.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import asyncio
+import logging
+from typing import Text, Optional
+
+import rasa.utils.io
+import rasa.train
+from examples.restaurantbot.policy import RestaurantPolicy
+from rasa.core.agent import Agent
+from rasa.core.interpreter import RasaNLUInterpreter
+from rasa.core.policies.memoization import MemoizationPolicy
+from rasa.core.policies.mapping_policy import MappingPolicy
+
+logger = logging.getLogger(__name__)
+
+
+async def parse(
+    text: Text, core_model_path: Text, nlu_model_path: Optional[Text] = None
+):
+    if nlu_model_path:
+        interpreter = RasaNLUInterpreter(nlu_model_path)
+    else:
+        interpreter = None
```

Comment:
Maybe I would instead use a RegexInterpreter and print a message like "No NLU model passed, parsing message using RegexInterpreter" or something like that so that you can still parse using the intent format